source/meta.tex

```diff
@@ -2236,43 +2252,45 @@
 forms the logical disjunction of its template type arguments.
 
 \pnum
+For the specialization \tcode{disjunction<>},
+let $\tcode{B}_{i}$ be \tcode{false_type}.
```

Comment:
```suggestion
```
Based on the addition of the explicit specialization to the synopsis, and that only the primary template is present in the `itemdecl` environment, I believe this is not necessary.